src/components/ProductList.jsx

```
@@ -19,14 +19,38 @@ function ProductList() {
     // TODO: DELETE API를 호출하고 fetchProducts() 호출
+    try {
+      await fetch("${API_URL}/${id}", {
+        method: "DELETE",
+      });
+      fetchProducts();
+    } catch (error) {
+      console.error("Error:", error);
+    }
   };
 
   const handleAdd = async (newProduct) => {
```

ESLint

[error] 'newProduct' is defined but never used.

(no-unused-vars)